services/src/main/resources/locale/portlet/chat/Resource_fr.properties

```diff
@@ -261,5 +261,5 @@ Notification.ChatMentionNotificationPlugin.label.footer=Si vous ne souhaitez pas
 UINotification.title.ChatMentionNotificationPlugin=Quelqu'un me mentionne dans le chat
 UINotification.label.ChatMentionNotificationPlugin=Mention dans le chat
 
-UINotification.title.ChatMessageReceivedNotificationPlugin=Quelqu'un m'envoie un message
+UINotification.title.ChatMessageReceivedNotificationPlugin=Je reçois un message dans une discussion chat
 UINotification.label.ChatMessageReceivedNotificationPlugin=Recevoir un message
```

Comment:
@hghorbel I find this not homogeneous : 
UINotification.title.ChatMentionNotificationPlugin=Someone mentions me in chat
UINotification.title.ChatMessageReceivedNotificationPlugin=I receive a message in a chat room

I think I prefer the previous formulation completing with "in a chat room": 
UINotification.title.ChatMessageReceivedNotificationPlugin=Someone sends me a message _in a chat room_
What do you think ?

Reply:
@rdenarie This is why i suggested the first sentence even if it was complicated to understand and formulate, we do have 2 use cases: 

- I receive a message in one to one discussion (which is a message to me indeed)
- I receive a message in a chat room I'm a member in (the message can be sent to me or to another user in the chat room).

We can just put **Someone sends me a message in a chat room** or just **Someone sends a message in a chat room** (without the word me) to conserve the multiple use cases. 

I prefer the second one without the word me as long as even the one to one discussion is considered as a chat room (Discussion). 
WDYT ?

Reply:
I propose : 

- Someone sends me a private message
- Someone sends a message in a chat room

Reply:
+1 for Someone sends a message in a chat room and we can change it whenever we want on crowdin.